meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/am64xx-sk/boot.cmd

```diff
@@ -0,0 +1,8 @@
+setenv loadaddr 0x90000000
+setenv devtype mmc
+setenv devnum ${mmcdev}
+run findfdt
+load ${devtype} ${devnum}:2 ${loadaddr} /boot/loader/uEnv.txt
```

Comment:
Do we really have /boot/loader/uEnv.txt with lmp-base? In theory that file is ostree specific.

Reply:
Dropped this recipe.  It wasn't needed and it was broken.